app/controllers/users_controller.rb

```diff
@@ -0,0 +1,3 @@
+class UsersController < ApplicationController
+  def show; end
+end
```

Comment:
- [ ] Good job adding show here, but please add index as well just as you did on posts_controller